features/azure/pkg.include

```diff
@@ -1,6 +1,10 @@
 # Azure images require waagent
-#waagent
+waagent
 # debootstrap does not pull python3-cffi-backend
-#python3-cffi-backend
-#
-#cloud-init
+python3-cffi-backend
+python3-cryptography
+python3-oauthlib
+cloud-init
+cloud-guest-utils
+hyperv-daemons
+isc-dhcp-client
```

Comment:
Well. Using ìsc-dhcp-client` will not work, as this OS uses `systemd-networkd`. Also it is not longer supported.